bitnami/wordpress-nginx/6/debian-12/rootfs/post-init.sh

```diff
@@ -10,16 +10,16 @@ set -o pipefail
 # set -o xtrace # Uncomment this line for debugging purposes
 
 # Only execute init scripts once
-if [[ ! -f "/bitnami/wordpress/.user_scripts_initialized" && -d "/docker-entrypoint-init.d" ]]; then
+if [[ ! -f "/bitnami/wordpress-nginx/.user_scripts_initialized" && -d "/docker-entrypoint-init.d" ]]; then
     read -r -a init_scripts <<< "$(find "/docker-entrypoint-init.d" -type f -print0 | sort -z | xargs -0)"
-    if [[ "${#init_scripts[@]}" -gt 0 ]] && [[ ! -f "/bitnami/wordpress/.user_scripts_initialized" ]]; then
-        mkdir -p "/bitnami/wordpress"
+    if [[ "${#init_scripts[@]}" -gt 0 ]] && [[ ! -f "/bitnami/wordpress-nginx/.user_scripts_initialized" ]]; then
+        mkdir -p "/bitnami/wordpress-nginx"
```

Comment:
Was this an intentional change? It causes an error when you start up with init scripts

`mkdir: cannot create directory '/bitnami/wordpress-nginx': Permission denied`

Reply:
cc @carrodher